core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -712,6 +746,184 @@ export class LoggedInUser extends BaseUser {
+      await this.clickOn(saveDraftButton);
+      await this.page.waitForSelector(toastMessage, {visible: true});
+      await this.page.waitForSelector(toastMessage, {visible: false});
+      await this.clickOn(explorationMobileDropdown);
+      await this.clickOn(publishExplorationMobileButton);
+    } else {
+      await this.clickOn(saveChangesButton);
+      await this.clickOn(saveDraftButton);
+      await this.page.waitForSelector(
+        `${publishExplorationButton}:not([disabled])`
+      );
+      await this.clickOn(publishExplorationButton);
+    }
+
+    await this.type(explorationTitleInput, 'Test Exploration');
+    await this.type(explorationGoalInput, 'Test Exploration');
+    await this.clickOn(explorationCategoryDropdown);
+    await this.clickOn('Algebra');
+    await this.clickOn(saveExplorationChangesButton);
+    await this.clickOn(explorationConfirmPublishButton);
+    await this.page.waitForSelector(explorationIdElement);
+    await this.page.click('.e2e-test-share-publish-close');
+    const explorationIdUrl = await this.page.$eval(
+      explorationIdElement,
+      element => element.textContent
+    );
+    return explorationIdUrl;
+  }
+
+  /**
+   * Function for navigating to the profile page for a given username.
+   */
+  async navigateToProfilePage(username: string): Promise<void> {
+    const profilePageUrl = `${profilePageUrlPrefix}/${username}`;
+    if (this.page.url() === profilePageUrl) {
+      return;
+    }
+    await this.goto(profilePageUrl);
+  }
+
+  /**
+   * Function to subscribe to a creator with the given username.
+   */
+  async subscribeToCreator(username: string): Promise<void> {
+    const profilePageUrl = `${profilePageUrlPrefix}/${username}`;
+
+    if (this.page.url() !== profilePageUrl) {
+      await this.navigateToProfilePage(username);
+    }
+
+    await this.clickOn(subscribeButton);
+    await this.page.waitForSelector(unsubscribeLabel);
+    showMessage(`Subscribed to the creator with username ${username}`);
+  }
+
+  /**
+   * This function checks if the number of subscribers equal to.
+   */
+  async expectNumberOfSubscribersToBe(number: number): Promise<void> {
+    const subscriberCount = await this.page.$eval(
+      '.e2e-test-oppia-total-subscribers',
+      element => element.textContent
+    );
+
+    if (subscriberCount && parseInt(subscriberCount) === number) {
+      showMessage(`Number of subscribers is equal to ${number}`);
+    } else {
+      throw new Error(`Number of subscribers is not equal to ${number}`);
+    }
+  }
+
+  /**
+   * Function for refreshing the page.
+   */
+  async refreshPage(): Promise<void> {
+    await this.page.reload();
+  }
+
+  /**
+   * Function for opening the subscribers tab.
+   */
+  async openSubscribersTab(): Promise<void> {
+    if (this.page.url() !== creatorDashboardUrl) {
+      await this.navigateToCreatorDashboardPage();
+    }
+
+    await this.page.click(subscriberTabButton);
```

Comment:
Just curious to know why we are using .click instead of `.clickOn`. Any specific reason?